iceoryx_posh/test/integrationtests/test_posh_mepoo.cpp

```diff
@@ -247,23 +242,42 @@ class Mepoo_IntegrationTest : public Test
         return true;
     }
 
-    void getMempoolInfoFromIntrospection(MemPoolInfoContainer& mempoolInfo)
+    void getMempoolInfoFromIntrospection(MemPoolInfoContainer& memPoolInfoContainer)
     {
-        auto currentUser = iox::posix::PosixUser::getUserOfCurrentProcess();
-        auto memoryManager = m_roudiEnv->m_roudiApp->m_mempoolIntrospection.m_segmentManager
-                                 ->getSegmentInformationWithWriteAccessForUser(currentUser.getName())
-                                 .m_memoryManager;
-        ASSERT_TRUE(memoryManager.has_value());
-        m_roudiEnv->m_roudiApp->m_mempoolIntrospection.copyMemPoolInfo(memoryManager.value().get(), mempoolInfo);
-
-        // internally, the chunks are adjusted to the additional management information;
-        // this needs to be subtracted to be able to compare to the configured sizes
-        for (auto& mempool : mempoolInfo)
+
+        iox::runtime::PoshRuntime::initRuntime(iox::roudi::INTROSPECTION_APP_NAME);
+
+        iox::popo::SubscriberOptions opts;
```

Comment:
Also just nitpicking but please don't use abbreviations when the name is not that long.